docs/administration/configure-project-quotas/_index.md

```diff
@@ -41,9 +44,9 @@ When setting project quotas, it is useful to know how Harbor calculates storage
   {{< note >}}
   When users push an image, the manifest is pushed last, after all of the associated blobs have been pushed successfully to the registry. If several images are pushed concurrently and if there is an insufficient number of tags left in the quota for all of them, images are accepted in the order that their manifests arrive. Consequently, an attempt to push an image might not be immediately rejected for exceeding the quota. This is because there was availability in the tag quota when the push was initiated, but by the time the manifest arrived the quota had been exhausted.
   {{< /note >}}
-- Shared blobs are only computed once per project. In Docker, blob sharing is defined globally. In Harbor, blob sharing is defined at the project level. As a consequence, overall storage usage can be greater than the actual disk capacity.
+- Shared blobs are computed only once per project. In Docker, blob sharing is defined globally. In Harbor, blob sharing is defined at the project level. As a consequence, overall storage usage can be greater than the actual disk capacity.
 - Retagging images reserves and releases resources: 
   -  If you retag an image within a project,  the storage usage does not change because there are no new blobs or manifests.
-  - If you retag an image from one project to another, the storage usage will increase.
+  - If you retag an image from one project to another, the storage usage increases.
```

Comment:
I don't think this is true?? retagging is just adding some metadata.  can you verify? @reasonerjt @steven-zou

Reply:
ok I can see it being interpreted both ways.  keep the edit for now